pySDC/projects/Second_orderSDC/stability_simulation.py

```diff
@@ -0,0 +1,287 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from pySDC.core.Errors import ProblemError
+from pySDC.core.Step import step
+
+from pySDC.projects.Second_orderSDC.plot_helper import set_fixed_plot_params
+from tabulate import tabulate
+class StabilityImplementation:
+    """
+    Routine to compute the stability domains of different configurations of SDC
```

Comment:
This is a class, not a routine. Because the name of the class is not very descriptive, I don't find it obvious what this is supposed to do.